runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -46,20 +40,27 @@ private MetricsUtils() {
     private static final String PROPERTY_LOG_INTERVAL = "corfu.metrics.log.interval";
     private static final String PROPERTY_METRICS_COLLECTION = "corfu.metrics.collection";
 
+    private static final long metricsCsvInterval;
+    private static final long metricsLogInterval;
+    private static final String metricsCsvFolder;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)